src/main/resources/static/assets/js/sdh.js

```diff
@@ -374,10 +369,12 @@ class ModalEditRow {
 						return review.domain === domain.text;
 					});
 					if (reviewForDomain.length === 0) {
-						return {id: '', domain: domain.text, last_reviewed: 'NA', clinician_priority:'None', patient_readiness: 'None', referred: this._selectText(this.yesNoOptions, false), referral_complete: this._selectText(this.yesNoOptions, false)};
+						return {id: '', domain: domain.text, last_reviewed: 'NA', clinician_priority:this.emptyOption, patient_readiness: this.emptyOption, referred: this._selectText(this.yesNoOptions, false), referral_complete: this._selectText(this.yesNoOptions, false)};
 					} else {
-						let review = reviewForDomain[0]; // Unique constraint prevents more than one
-						return {id: review.id, domain: domain.text, last_reviewed: this._formatDate(review.updatedAt), clinician_priority: review.clinicianPriority, patient_readiness: review.patientReadiness, referred: this._selectText(this.yesNoOptions, review.referred), referral_complete: this._selectText(this.yesNoOptions, review.referralComplete)};						
+						const review = reviewForDomain[0]; // Unique constraint prevents more than one
+						const priority = (review.clinicianPriority != null) ? review.clinicianPriority.description : this.emptyOption;
+						const readiness = (review.patientReadiness != null) ? review.patientReadiness.description : this.emptyOption;
```

Comment:
It would be more idiomatic to just use truthiness here:

```js
const priority = review.clinicianPriority ? review.clinicianPriority.description : this.emptyOption;
```